plugins/examples/pod-counter/src/index.tsx

```diff
@@ -9,11 +9,19 @@ function PodCounter() {
 
 registerAppBarAction(PodCounter);
 
-// registerAppBarAction({
-//   processor: (_, actions) => {
-//     const newActions = actions.filter(action => action.id !== 'pod-counter')
-//     newActions.push({action: <PodCounter />, id: 'pod-counter'})
-//     return newActions;
-//   },
-//   id: 'pod-counter-processor',
-// });
+// We can also reorder the actions in the app bar.
+registerAppBarAction((_, actions) => {
+  // Remove the notifications action button
+  const newActions = actions.filter(action => action.id !== DefaultAppBarAction.NOTIFICATION);
+
+  // Add an extra pod counter action button
+  // newActions.push({action: <PodCounter />, id: 'pod-counter
```

Comment:
Is this intentionally left here? (the code comment is not finished).
Maybe we could add a label or something as an example here.